bot/resources/tags/intents.md

```diff
@@ -0,0 +1,19 @@
+**Using intents in discord.py**
+
+Intents are a feature of Discord that tells the gateway exactly which events to send your bot. By default, discord.py has all intents enabled, except for the `Members` and `Presences` intents, which are needed for events such as `on_member` and to get members' statuses.
+
+To enable one of these intents, you need to first go to the [Discord developer portal](https://discord.com/developers/applications), then to the bot page of your bot's application. Scroll down to the `Privileged Gateway Intents` section, then enable the intents that you need.
```

Comment:
Is it worth linking to the [intents section of the Discord docs](https://discord.com/developers/docs/topics/gateway#gateway-intents) for a bit more info?

Reply:
> Is it worth linking to the [intents section of the Discord docs](https://discord.com/developers/docs/topics/gateway#gateway-intents) for a bit more info?

That seems like it would be useful. The Discord dev docs explain it rather well.